tell/install_supplement.py

```diff
@@ -72,10 +72,11 @@ def fetch_zenodo(self):
                         # construct temporary file full path with name
                         tfile = os.path.join(tdir, f)
 
-                        print(f"Unzipped: {out_file}")
```

Comment:
The sheer number of files being unzipped was crashing my Jupyter notebook, so i removed this in favor of a "Done!" at the end.

Reply:
perfect, that seems good to me!